setup.py

```diff
@@ -18,7 +18,10 @@
         "apache-libcloud==3.1.0",
         "cryptography==38.0.1",
         "docopt==0.6.2",
-        "gevent==20.9.0",
+        # https://pypi.org/project/gevent/ the latest version resolves problem for Mac M1 chips
+        # This issue is caused by a program attempting to load an x86_64-only library from a native arm64 process.
+        # More https://stackoverflow.com/questions/71443345/gevent-cant-be-installed-on-m1-mac-using-poetry
+        "gevent==21.12.0",
```

Comment:
Can you please create separate PR for this file change of pinning specific version of dependencies?